core/utils.py

```diff
@@ -94,6 +94,17 @@ class ExplorationConversionError(Exception):
     pass
 
 
+class ThrowGetError:
+    """Error class to throw error while invoking the get function."""
+
+    def __init__(self, message: str):
+        self.err_message = message
+
+    def get(self, *args: Any, **kwargs: Any) -> None:
+        """Get function throws error when invoked."""
+        raise ValueError(self.err_message)
+
+
```

Comment:
I'm bit unsure about the naming here, also as we only use it in the handler schema validation should we just move it there?

Reply:
Yeah, I have changed the name to `RaiseErrorOnGet` and moved it to `base.py`